scalableminds/webknossos
# Disable AI analysis for 2D and ND datasets

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- test that "AI Analysis" button is disabled if dataset is in 2D, ND or only has color layers in uInt24 data format
- make sure that you cannot select color layers with dataformat uint24 in layer selection (neither train nor run)
- everything else should work as before :)

### TODOs:
- [x] disable for 2D and ND
- [x] error for weird units
- [x] maybe disable/filter out uint24 layer for train model?
- [x] find out whether uint24 blacklisted or uint8 whitelisted
- [x] test for 2D
- [x] test review changes, e.g. with DS without color layer


### Issues:
- fixes #7953

------
(Please delete unneeded items, merge only when none are left open)
- [ ] Removed dev-only changes like prints and application.conf edits :exclamation: :exclamation: :boom: :boom:
